src/app/(app)/(home)/search-filters/categories-sidebar.tsx

```
@@ -12,59 +12,66 @@ import {
-                      style={{ backgroundColor: `${cat.color}` }}
+        {isLoading || !data ? (
+          <p>Loading...</p>
+        ) : (
+          <NavigationMenu
+            viewport={false}
+            className="items-start max-w-full [&>div]:min-w-full"
+          >
+            <NavigationMenuList className="flex-col">
+              {data.map((cat: any) => (
+                <NavigationMenuItem className="w-full" key={cat.id}>
+                  {cat.subcategories && cat.subcategories.length ? (
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)